### PR TITLE
Misc fixes

### PR DESCRIPTION
**Pull request by bangert**
I required a fix similar to the first commit in this PR to make it work with Keycloak.

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
   "version": "0.0.6",
+  "hompage": "https://github.com/el-davo/ng-oauth2",
   "license": "MIT",
   "main": [
     "dist/ng-oauth2.min.js"
   ],
   "dependencies": {
-    "angular": "1.3.x",
-    "angular-mocks": "1.3.x",
+    "angular": "^1.3.x",
```
Comment (el-davo) [inline bower.json]: Doesnt the x do the exact same thing?

Comment (el-davo) [inline bower.json:11]: Is this a bit dangerous to be allowing all major releases?

File: bower.json [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
+    "angular-mocks": "^1.3.x",
```
Comment (el-davo) [inline bower.json:12]: I think these changes are causing travis to fail?

https://travis-ci.org/el-davo/ng-oauth2/builds/114203533

File: bower.json (hunk 1 above)
Comment (bangert) [inline bower.json:11]: it seems to work fine here on angular-1.5.

Comment (el-davo) [inline bower.json:11]: Awesome, I guess this is ok then. Make sure you fix the conflicts

File: bower.json (hunk 2 above)
Comment (bangert) [inline bower.json:12]: yeah - for some reason tests don't work over here.
It was originally not my intention to have all 4 changes in one pull request. will do a better job next time.

Comment (el-davo) [inline bower.json:12]: No worries. It looks like it just needs a resolution in bower.json. Try deleting you bower folder and then bower install again. It should ask you for a resolution. Give it an anwer and put a ! after your answer to preserve it

**Comment by bangert** [comment]
these are the things i needed to do to make testing work on my end. could i ask you to double check this?

as for the still failing CI checks: could it be that an old bower version is lingering around in the cached node_modules directory? If so, could you try to clear the travis-ci cache?
(i am new to much of this, please bear with me)

**Comment by el-davo** [comment]
Thanks, This is merged. I bumped to version 1.0.0 with your changes. Let me know if it works on your end when you include it

**Comment by bangert** [comment]
It works great. Thanks.

**Pull request merged**